### ISSUE TITLE
Support for Onedrive videos

### ISSUE THREAD

**Issue by algorithmor** [nathancy/jekyll-embed-video#8]
Hi Nathancy, 
Thanks for creating this helpful project. Would it be possible to add support for embedding video files from Onedrive easily. I think that's the only major source missing right now.

**Comment by nathancy** [comment]
I agree that would be a great addition. I'm sure it's possible to embed
OneDrive videos but unfortunately I don't have a OneDrive account so I
can't test it myself. If you happen to find a way, feel free to open a pull
request and I can add it in! Thanks
